bin/lint-migrations-file/src/lint_migrations_file.clj

```diff
@@ -77,15 +71,11 @@
   [change-log]
   (let [problem-cols (atom [])
         walk-fn      (partial assert-no-types-in-change-set #{"blob" "text"} problem-cols)]
```

Comment:
Removing the postwalk without also removing this state is leaving behind half-refactored code. If you're going to refactor `assert-no-types-in-change-set` you can probably go all the way and remove the need for this `problem-cols` atom. The use of `assert-no-types-in-change-set` can be simplified to a `filter` on changeset maps. It's a pure operation and doesn't need external state.